src/main/webapp/app/complaints/complaints.component.html

```diff
@@ -2,57 +2,80 @@
             </p>
 
             <p *ngIf="alreadySubmitted" class="mt-4">
-                {{ 'artemisApp.complaint.alreadySubmitted' | translate }} {{ submittedDate | date: 'MMM d, y HH:mm:ss' }}:
-                <span *ngIf="handled && accepted" class="text-light bg-success"> &nbsp;{{ 'artemisApp.complaint.acceptedLong' | translate }}&nbsp; </span>
+                {{
+                    complaintType === ComplaintType.MORE_FEEDBACK ? ('artemisApp.moreFeedback.alreadySubmitted' | translate) : ('artemisApp.complaint.alreadySubmitted' | translate)
+                }}
+                {{ submittedDate | date: 'MMM d, y HH:mm:ss' }}
+                <span *ngIf="handled && accepted" class="text-light bg-success">
+                    &nbsp;{{
+                        complaintType === ComplaintType.MORE_FEEDBACK ? ('artemisApp.moreFeedback.acceptedLong' | translate) : ('artemisApp.complaint.acceptedLong' | translate)
+                    }}&nbsp;
+                </span>
                 <span *ngIf="handled && !accepted" class="text-light bg-danger"> &nbsp;{{ 'artemisApp.complaint.rejectedLong' | translate }}&nbsp; </span>
             </p>
 
-            <textarea class="col-12 px-1" rows="4" maxLength="2000" [(ngModel)]="complaintText" [readOnly]="alreadySubmitted" [disabled]="alreadySubmitted"></textarea>
+            <textarea
+                id="complainTextArea"
+                class="col-12 px-1"
+                rows="4"
+                maxLength="2000"
+                [(ngModel)]="complaintText"
+                [readOnly]="alreadySubmitted"
+                [disabled]="alreadySubmitted"
+            ></textarea>
 
             <div class="row" *ngIf="!alreadySubmitted">
                 <div class="col-6">
-                    <button class="btn btn-primary" [disabled]="complaintText.length <= 0" (click)="createComplaint()">{{ 'artemisApp.complaint.submit' | translate }}</button>
+                    <button class="btn btn-primary" [disabled]="complaintText.length <= 0" (click)="createComplaint()">
+                        {{ complaintType === ComplaintType.MORE_FEEDBACK ? ('artemisApp.moreFeedback.button' | translate) : ('artemisApp.complaint.submit' | translate) }}
+                    </button>
                 </div>
             </div>
         </div>
 
         <div class="col-12 col-md-6">
             <p class="col-12" *ngIf="complaintResponse" class="mt-4">
-                {{ 'artemisApp.complaint.responseExists' | translate }} {{ complaintResponse.submittedTime | date: 'MMM d, y HH:mm:ss' }}:<br />
+                {{ complaintType === ComplaintType.MORE_FEEDBACK ? ('artemisApp.moreFeedback.responseExists' | translate) : ('artemisApp.complaint.responseExists' | translate) }}
+                {{ complaintResponse.submittedTime | date: 'MMM d, y HH:mm:ss' }}:<br />
```

Comment:
I think we should use the moment API here

Reply:
To be honest I don't understand why we need to use moment API for date formatting here. Can you explain, please?

Reply:
Because we use it in other places as well and it is easier for the users to understand that something was 1 min, 1 hour, 1 day or 1 month ago than focusing on one specific date format. In addition, this typically allows to see the actual date string when hovering over the string that is displayed by the moment API.

Also it is quite bad to define the date format explicitly here, because then it can get inconsistent with other places and it can get very hard to find all places and make it consistent.

Can you please change to the moment API?

Reply:
Found that we have amTimeAgo pipe and used it here 
![Screenshot_2019-07-31 Modeling Exercises](https://user-images.githubusercontent.com/17262376/62173839-cfa85700-b337-11e9-8c0e-97125af4fa97.png)